client/src/views/home.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Box,
+  Container,
+  Divider,
+  Heading,
+  Stack,
+  Stat,
+  StatLabel,
+  StatNumber,
+  Text,
+} from "@liftedinit/ui";
+
+export function Home() {
+  return (
+    <Container maxW="container.md">
+      <Heading my={6}>Talib</Heading>
+      <Text>A Block Explorer for the Many Protocol</Text>
+      <Divider my={6} />
+      <Stack direction="row" mt={6}>
+        <Stat bg="white" p={6}>
+          <StatLabel>Block Height</StatLabel>
+          <StatNumber>
+            <pre>[block height]</pre>
+          </StatNumber>
+        </Stat>
```

Comment:
I'll pull this out into a shared component once I've got it looking right. Just repeating them for now.